### PR TITLE
Add bedrock server support

### PR DESCRIPTION
**Pull request by Iapetus-11**
Code for checking the status of a Minecraft BE server has been added, but is commented out as a dependency doesn't support Python 3.5. I also wasn't sure whether to add support for Python 2.x as it seemed like it was no longer being supported by this project.

Let me know if I need to make any tweaks or changes!

Example usage:
```py
import mcstatus

bedrock_server = mcstatus.MinecraftBedrockServer.lookup('xenonmc.ml')
response = bedrock_server.status()

print(response.latency)
print(response.version.protocol)
print(response.version.brand)
print(response.motd)
print(response.players_max)
print(response.players_online)
print(response.map)
print(response.gamemode)
```

### CODE REVIEW

**Comment by Iapetus-11** [comment]
Looks like the module I used for udp support isn't supported by Python 3.5. I guess I'll work on a solution unless the maintainer wants to remove support for Python 3.5.

**Comment by Iapetus-11** [comment]
I figured out + implemented regular (non-async) support for getting the status of Bedrock Edition servers, and commented out the async methods + removed `asyncio-dgram` from the `requirements.txt`. It should now be compatible with Python 3.5-3.9!

**Comment by kevinkjt2000** [comment]
Thanks for taking the time to do this! Would you mind adding some unit tests under `mcstatus/tests/` directory? To run the tests in the same way the release automation does, feel free to follow along with these highlighted commands https://github.com/Dinnerbone/mcstatus/blob/c2fb996528c64d6e84ea72e6f438e8ef02ac6a64/release.sh#L4-L12

**Comment by kevinkjt2000** [comment]
Please don't bother with python2 support. This library is only promising python3 support as of v3.0.0 https://github.com/Dinnerbone/mcstatus/commit/db85166ad540cb52e1613a660a18e18a97808fa0

**Comment by Iapetus-11** [comment]
Unit tests were added, and as `asyncio-dgram` now supports Python 3.5 async Bedrock Edition status was also added back.

**Comment by kevinkjt2000** [comment]
I’ll volunteer some time tomorrow to look at this.

**Comment by Iapetus-11** [comment]
It might need some restructuring as I wasn't exactly sure where to put everything, but I'd be happy to do it!

File: mcstatus/tests/test_bedrock_status.py
Comment (kevinkjt2000) [inline mcstatus/tests/test_bedrock_status.py:25]: https://docs.python.org/3/library/unittest.mock.html has a `patch` function that would be able to mock `perf_counter` and `socket` stuff. No need to do this though. In the future, I'll get bored and revisit this after adding test coverage. There is not much to gain from unit tests testing that `socket` is working as expected.

Please delete this empty test function for now.

**Review by kevinkjt2000 — CHANGES_REQUESTED** [review]
Thanks again for contributing and thanks for being patient while I slowly got around to reviewing this. Only a few comments to address before I can approve.

File: mcstatus/bedrock_status.py
Comment (Iapetus-11) [reply mcstatus/bedrock_status.py:7]: I don't think it's similar enough, mainly due to that one uses TCP and the other uses UDP. However for consistency and clarity we could rename the file to `bedrock_pinger.py` and `BedrockServerStatus` to `BedrockServerPinger`?

Comment (kevinkjt2000) [reply mcstatus/bedrock_status.py:7]: Maybe I'm splitting hairs here since these are basically internal classes that are not exported at the top-level `__init__.py` file. On further thinking about this, I'm okay with it as is.

**Review by kevinkjt2000 — APPROVED** [review]

**Pull request merged**
